CheckersCheckpoint/Game.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace CheckersCheckpoint
+{
+    public class Game
+    {
+        public Game()
+        {
+            Board board = new Board();
+            board.GenerateCheckers();
+            board.DrawBoard();
+
+            Console.WriteLine("\nIf you want to move a checker one space diagonally forward, enter 'move'.");
+            Console.WriteLine("\nIf a jump is available for one of your checker, you must enter 'jump'.");
+
+            string choice = Console.ReadLine();
+
+            do
+            {
+                switch (choice)
+                {
+                    case "move":
+
+                        Console.WriteLine("Enter checker Row to move:");
+                        int row = int.Parse(Console.ReadLine());
+                        Console.WriteLine("Enter checker Column:");
+                        int column = int.Parse(Console.ReadLine());
```

Comment:
Every time you use just Parse and not TryParse a QA person cries 😭

You should be doing input validation (checking what the user typed in matches what is acceptable).

Reply:
i think you graded the wrong thing, that was my mockup.